### ISSUE TITLE
Error on importing the libraries

### ISSUE THREAD

**Issue by jaballogian** [tf2pickup-org/mumble-client#233]
### Description

Error after importing the `Channel` or `Client` object from this library as seen in the screenshot below
![image](https://user-images.githubusercontent.com/24468466/212839693-a6b192b8-9d9e-4e9d-8fb7-03bfb58eb8d0.png)

```cmd
Compiled with problems:X

ERROR in ./node_modules/@tf2pickup-org/mumble-client/dist/client.js 22:13-26

Module not found: Error: Can't resolve 'os' in 'D:\A.ing\Works\Others\Lacak-and-Valid\Exercises\React-Js\learn-mumble-client\node_modules\@tf2pickup-org\mumble-client\dist'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "os": require.resolve("os-browserify/browser") }'
	- install 'os-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.fallback: { "os": false }


ERROR in ./node_modules/@tf2pickup-org/mumble-client/dist/tls-connect.js 12:30-44

Module not found: Error: Can't resolve 'tls' in 'D:\A.ing\Works\Others\Lacak-and-Valid\Exercises\React-Js\learn-mumble-client\node_modules\@tf2pickup-org\mumble-client\dist'
```

### Steps to produce
- create a React JS app using the `npx create-react-app` command
- enter the React Js app and install the `@tf2pickup-org/mumble-client` dependency by using the `npm i @tf2pickup-org/mumble-client` command
- go to the `App.js` file and add `import { Channel, Client } from "@tf2pickup-org/mumble-client";`
- run the React Js app by using the `npm start` command
- the error appears

**Comment by garrappachc** [comment]
[Problematic line](https://github.com/tf2pickup-org/mumble-client/blob/8abb9285e8e90bd9355575605ac28df99d234fb5/src/client.ts#L34) - you're trying to use mumble-client in the browser, and the `os` module exists only in node.js environment.
I'll have it fixed.

**Comment by jaballogian** [comment]
> [Problematic line](https://github.com/tf2pickup-org/mumble-client/blob/8abb9285e8e90bd9355575605ac28df99d234fb5/src/client.ts#L34) - you're trying to use mumble-client in the browser, and the `os` module exists only in node.js environment. I'll have it fixed.

You're right. That's why there is no UI/frontend/HTML file on this repository.

I thought this library could be used on the frontend side because there was a "client" name on the repository title ("mumble-client").

**Comment by garrappachc** [comment]
> > [Problematic line](https://github.com/tf2pickup-org/mumble-client/blob/8abb9285e8e90bd9355575605ac28df99d234fb5/src/client.ts#L34) - you're trying to use mumble-client in the browser, and the `os` module exists only in node.js environment. I'll have it fixed.
> 
> You're right. That's why there is no UI/frontend/HTML file on this repository.
> 
> I thought this library could be used on the frontend side because there was a "client" name on the repository title ("mumble-client").

It is a mumble client, as in it connects to a remote mumble server. It was intented to run on Node.JS environment primarily (my use-case), but there is no reason why it couldn't run in the browser as well.
